demisto_sdk/commands/validate/validators/IF_validators/IF119_select_values_cannot_contain_multiple_or_only_empty_values_in_single_select_types.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.common.constants import GitStatuses, IncidentFieldType
+from demisto_sdk.commands.content_graph.objects.incident_field import IncidentField
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    FixResult,
+    ValidationResult,
+)
+
+ContentTypes = IncidentField
+
+
+def select_values_do_not_contain_multiple_or_only_empty_values_in_single_select_types(
+    content_item: ContentTypes,
+) -> bool:
+    if content_item.data.get("type") == IncidentFieldType.SINGLE_SELECT:
+        select_values = content_item.data.get("selectValues") or []
+        empty_string_count = sum(select_value == "" for select_value in select_values)
+        if empty_string_count > 1 or (
+            len(select_values) == 1 and empty_string_count == 1
+        ):
+            return False
+    return True
+
+
+class SelectValuesCannotContainMultipleOrOnlyEmptyValuesInSingleSelectTypesValidator(
+    BaseValidator[ContentTypes]
+):
+    error_code = "IF119"
+    description = "We do not allow for incidentFields with singleSelect types to have in the selectValues more than one or only emtpy option"
+    rationale = "Due to UI issues, we cannot allow more than one or only empty values for selectValues field"
+    error_message = "singleSelect types cannot contain more than one or only empty values in the selectValues field."
```

Comment:
Let's make sure only the relevant issue appears, maybe consider returning the relevant error msg rather than True/False from select_values_do_not_contain_multiple_or_only_empty_values_in_single_select_types.